src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_Window.cs

```diff
@@ -17,6 +17,16 @@ public void When_CreateNewWindow()
 			// This used to crash on wasm which was trying to create a second D&D extension
 			var sut = new Window();
 		}
+#endif
+#if __SKIA__
+		[TestMethod]
```

Comment:
This test will not pass, as runtime tests are run in the scope of Samples app and I think the unit test runner there actually changes title to the name of the current test. Instead you can add that assert here https://github.com/unoplatform/uno/blob/37323c22809fc0b1478acafa64f854c5fba85818/src/SamplesApp/SamplesApp.Shared/App.xaml.cs#L74

This is run immediately after startup, so the title is still the "original" one there. See how the `AssertIssue1790()` method is formed, this could be similar

Reply:
Thank you a lot!!! 
probably this will solve not only this one, but another failed PR that i have!